packages/hardhat-core/test/internal/hardhat-network/provider/modules/hardhat.ts

```diff
@@ -140,6 +140,34 @@ describe("Hardhat module", function () {
             impersonatedAddress
           );
         });
+
+        it.only("lets you impresonate a contract", async function () {
```

Comment:
Probably want to remove this .only when pushing.

Reply:
Good catch @asafyish. Fixed.